src/flypad-backend/src/FlyPadBackend.cpp

```diff
@@ -134,18 +183,38 @@ void FlyPadBackend::simConnectProcessMessages() {
 void FlyPadBackend::simConnectProcessSimObjectData(const SIMCONNECT_RECV_SIMOBJECT_DATA* data) {
   // process depending on request id from SimConnect_RequestDataOnSimObject()
   switch (data->dwRequestID) {
-    case 0:
+    case DataStructureRequestIDs::SimulationDataRequestID:
       // store aircraft data in local data structure
       simulationData = *((SimulationData*) &data->dwData);
       return;
 
-    case 1:
+    case DataStructureRequestIDs::PushbackDataRequestID:
       // store aircraft data in local data structure
       pushbackData = *((PushbackData*) &data->dwData);
       return;
 
     default:
-      cout << "FLYPAD_BACKEND: Unknown request id in SimConnect connection: ";
+      cout << "FLYPAD_BACKEND: Unknown request id in simConnectProcessSimObjectData(): ";
+      cout << data->dwRequestID << endl;
+      return;
+  }
+}
+
+void FlyPadBackend::simConnectProcessClientData(const SIMCONNECT_RECV_CLIENT_DATA* data) {
+  // process depending on request id from SimConnect_RequestClientData()
+  switch (data->dwRequestID) {
+    case DataStructureRequestIDs::AllIVAORequestID:
+      IVAOData = new ThirdPartyDataIVAO();
+      *IVAOData = *((ThirdPartyDataIVAO*) &data->dwData);
```

Comment:
Is this why you use a raw pointer? No other way?

Reply:
I've used a raw pointer to be able to know if there's new data without having the need to test previous values